app/views/words/new.html.slim

```diff
@@ -0,0 +1,13 @@
+.container
+  .row
+    .col-sm-9.col-md-7.col-lg-5.mx-auto
+      h2.mt-5 Import from CSV-Excel file
+      br
+      = form_for @file, multipart: true do |f|
+        .form-group
+          = f.label :project
+          = f.select :project_ids, options_for_select(@word_project), {}, class: "form-control"
```

Comment:
Neu import khong chon project thi se the nao

Reply:
vẫn post được bth ạ